packages/design-system/src/components/Dialog/Dialog.tsx

```diff
@@ -1,49 +1,170 @@
 import { EVENT_CATEGORY, MAX_LENGTH, sendLinkEvent } from '../analytics/SendAnalytics';
 import AriaModal from 'react-aria-modal';
```

Comment:
I can't find the type definitions for this package. I'm not sure how there are no type definitions and TypeScript isn't complaining about it.